HelpSource/Classes/SimpleNumber.schelp

```diff
@@ -368,16 +368,16 @@ method:: asAudioRateInput
 Converts this into an audiorate input.
 
 method:: asTimeString
-Produces a time string in the format code::ddd:hh:mm:ss.sss::, interpreting the receiver as time in
-seconds. See link::Classes/String#-asSecs:: for the inverse function.
+Produces a time string in the clock format inspired by ISO 8601 time interval display (truncated representation) code::ddd:hh:mm:ss.sss::, interpreting the receiver as time in seconds. See link::Classes/String#-asSecs:: for the inverse function.
 argument:: precision
-accuracy of the millisecond format; the string will always be formatted with 3 decimal places for
-milliseconds. Minimum value: 0.001.
+accuracy of the fraction of seconds; since the number of decimal places is also an argument, minimum value of code::precision:: is code::10.pow(decimalPlaces.neg)::, i.e. code::0.001:: for code::decimalPlaces = 3::
```

Comment:
this kinda makes it sound like values under that minimum are not allowed -- could you reword it to clarify that those values will be replaced with the minimum (or clamped to it)?